doc/source/Platforms.rst

```diff
@@ -351,6 +352,16 @@ This instructions are meant to be a reference that users can follow to set up th
    # Note - need to pin to version 5
    brew install qt@5.15.3
 
+.. warning::
```

Comment:
Should we just say `brew install qt@5` instead in line 353?

Reply:
Made this change.